src/index.tsx

```diff
@@ -85,4 +85,4 @@ function SquircleBackground({
 }
 
 export { SquircleView }
-export type { SquircleParams }
+export type { SquircleParams, SquircleViewProps }
```

Comment:
This is reasonable but unrelated to this PR, could you create a separated PR for this?